index.cjs.js

```diff
@@ -0,0 +1,747 @@
+// DO NOT edit this file, it's auto generate from 'constructor/build.js', any changes will be overwrite. 
```

Comment:
Should this file be in `.gitignore` since it is generated?

Reply:
No, `index.cjs.js`(CommonJS Module) is built for compatibility reasons, in case some users are prefer to use `require` syntax. Unless we decide not to support commonjs anymore, otherwise we should publish it with `index.js`(ESModule) together.

Or use another way, we can built the `index.cjs.js` during the npm installation by using `"scripts" : { "postinstall": "node -r esm constructor/build.js" }` in the `package.json`. But I think this is unnecessary.